extras/python/fogbench/__main__.py

```diff
@@ -172,15 +173,25 @@ def read_out_file(_file=None, _keep=False, _iterations=1, _interval=0):
         # Pre-calculate the messages and size
         msg_transferred_itr = 0  # Messages transferred in every iteration
         byte_transferred_itr = 0  # Bytes transferred in every iteration
+
+        # Start sending the messages to server
+        _start_time.append(datetime.now())  # Start time of every iteration
+
         for idx, r in enumerate(readings_list):
             readings_list[idx]["key"] = str(uuid.uuid4())
+
+            is_sent = False
+            if send_to == 'coap':
```

Comment:
This if must control the loops, i.e. the if will then allow a loop to coap or http on condition.

Reply:
@izoratti  please check https://github.com/foglamp/FogLAMP/pull/541 do we want the fix it *(back to) like that?!